src/Microsoft.InsightsGenerator/RulesEngine.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
```

Comment:
please add MIT license header block to this file similar to other files in repo.